app/services/finance/business_case_summary/summarizer.rb

```diff
@@ -27,8 +27,12 @@ def stakeholder
         @financial_rule.keys.first
       end
 
+      def incoming
```

Comment:
I think it's confusing to have an instance variable called `@incoming` and an instance method also called `incoming`. Could the latter be renamed to something more meaningful? `incoming_from_rule` or something like that?